src/js/views/patients/worklist/worklist_views.js

```diff
@@ -222,6 +223,43 @@ const EmptyFindInListView = View.extend({
   `,
 });
 
+const CountView = View.extend({
```

Comment:
could see a world where this view is in some `patient/shared` directory.  I know the flow doesn't apply everywhere, but if that flag isn't set, it isn't set..  otherwise it is all identical correct?

Reply:
I pushed the other changes up, but I'm still working on this.